### ISSUE TITLE
Typo `torch.cat` should be `torch.stack`

### ISSUE THREAD

**Issue by WenjieDu** [WenjieDu/BenchPOTS#22]
### 1. System Info

benchpots v0.3

### 2. Information

- [x] The official example scripts
- [ ] My own created scripts

### 3. Reproduction

https://github.com/WenjieDu/BenchPOTS/blob/f7d4c97d47c20e19adfff411a7fe4dcd95fb23b3/benchpots/utils/sliding.py#L84

`torch.cat` should be `torch.stack`, otherwise cannot stack the samples into a new tensor with one more dim

### 4. Expected behavior

/
